docs/sources/operators-guide/getting-started-helm-charts/_index.md

```diff
@@ -0,0 +1,194 @@
+   An ingress enables you to externally access a Kubernetes cluster via the hostname defined by the _`<ingress-host>`_ variable.
+   Replace _`<ingress-host>`_ with a suitable hostname that DNS can resolve to the external IP address of the Kubernetes cluster. For more information, see [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/).
+   
+   Without using an ingress, it is still possible to access Grafana Mimir from inside the cluster. In such case, replace _`<ingress-host>`_ with _`<release-name>`_`-mimir-nginx.`_`<namespace>`_`.svc:80` throughout the rest of the procedure.
+
+1. Install Grafana Mimir using the Helm chart:
+
+   ```console
+   helm -n <namespace> install <release-name> grafana/mimir-distributed -f custom.yaml
+   ```
+
+1. Check the statuses of the Mimir services:
+
+   ```console
+   kubectl -n <namespace> get pod
+   ```
+
+   The results look similar to this:
+
+   ```console
+   kubectl -n dev get pod
+   NAME                                                       READY   STATUS      RESTARTS   AGE
+   <release-name>-mimir-nginx-69fd969c-g6bkn                  1/1     Running     0          159m
+   <release-name>-minio-5757f44456-dz7xx                      1/1     Running     0          159m
+   <release-name>-mimir-distributed-make-bucket-job-f5gbj     0/1     Completed   0          159m
+   <release-name>-mimir-distributor-7db5f7c8bc-444sd          1/1     Running     0          148m
+   <release-name>-mimir-overrides-exporter-79b475c98d-sqzxx   1/1     Running     0          148m
+   <release-name>-mimir-querier-5cb58b7b9c-zwnr6              1/1     Running     0          148m
+   <release-name>-mimir-query-frontend-74d666bcd5-4dcwh       1/1     Running     0          148m
+   <release-name>-mimir-ruler-745fff6c6d-8swsv                1/1     Running     0          148m
+   <release-name>-mimir-alertmanager-0                        1/1     Running     0          148m
+   <release-name>-mimir-compactor-0                           1/1     Running     0          148m
+   <release-name>-mimir-ingester-1                            1/1     Running     0          147m
+   <release-name>-mimir-ingester-2                            1/1     Running     0          147m
+   <release-name>-mimir-store-gateway-0                       1/1     Running     0          146m
+   <release-name>-mimir-ingester-0                            1/1     Running     0          144m
+   ```
+
+1. What until all of the pods have a status of `Running` or `Completed`, which might take a few minutes.
+
+## Configure Prometheus to write to Grafana Mimir
+
+Add the following YAML snippet to your Prometheus configuration file and restart the Prometheus server:
+
+```yaml
+remote_write:
+  - url: http://<ingress-host>/api/v1/push
+```
+
+The configuration for a Prometheus server that scrapes itself and writes those metrics to Grafana Mimir looks similar to this:
+
+```yaml
+remote_write:
+  - url: http://<ingress-host>/api/v1/push
```

Comment:
Using ingress-host will only work when Prometheus runs outside of Kubernetes server. Do we assume such setup?

Reply:
We're trying to cover both cases, but maybe it's too ambitious . There's a note further up that says what URL to use if you don't have ingress.

Reply:
As discussed in real life, we'll have two chapters for (outside the cluster with ingress) and (inside the cluster using metamonitoring to loop back on itself) CC: @osg-grafana  @dimitarvdimitrov 

Should be a pretty cool and self contained way. We can skip Prometheus , or just mention that the same kind of remote write setup works for it.